File: FormalBench-data/FilteredData/specifications/seed_726_per_category_10_653ade686f/c/DiameterCircle.c
#include "java_arrays.h"
#include <stdint.h>

/*@
  assigns \nothing;
  ensures \result == (int32_t)((integer)left * right);
*/
static int32_t java_mul(int32_t left, int32_t right) {
    return (int32_t)((uint32_t)left * (uint32_t)right);
}

/*@
  assigns \nothing;
  ensures \result == (int32_t)(2 * (integer)r);
*/
int32_t diameterCircle(int32_t r) {
    return java_mul(2, r);
}

File: FormalBench-data/FilteredData/specifications/seed_726_per_category_10_653ade686f/java/DiameterCircle.java
import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;


class DiameterCircle {
    
    /*@ public normal_behavior
      @ assignable \nothing;
      @ ensures \result == \java_math(2 * r);
      @*/
    //@ code_java_math
    public static int diameterCircle(int r) {
        return 2 * r;
    }
}

